### DISCUSSION TITLE
Turning monolithic Shiny app into a microservices-based structure

### DISCUSSION THREAD

**Discussion by yohanespm77** [teamalgoritma/community#697] [Q&A] [answered]
Apakah ada referensi untuk mengubah monolitik shiny menjadi mikroservis ?

Mohon bantuannya kak, terima kasih 🙏

**Comment by kinantytasyao** [comment, marked as answer]
Dear @yohanespm77 

Berikut referensi yang dapat kami berikan terkait turning monolithic Shiny app into a microservices-based structure:

- https://youtu.be/pzKkdadIgFU
- https://github.com/tamilyn/shiny-microservice-demo

Demikian, semoga dapat membantu :)

**Reply by yohanespm77** [reply]
Baik, terima kasih infonya mba.
